logstash-core/src/main/java/org/logstash/Cloner.java

```diff
@@ -1,5 +1,8 @@
 package org.logstash;
 
+import org.jruby.RubyNumeric;
```

Comment:
@andrewvc this one looks like a dead import, you didn't add any `RubyNumeric` usage?